apps/cowswap-frontend/src/modules/usdAmount/updaters/UsdPricesUpdater.ts

```diff
@@ -52,11 +50,7 @@ export function UsdPricesUpdater() {
 
       setUsdPricesLoading(debouncedQueue)
 
-      return processQueue(debouncedQueue, getUsdcPrice).catch((error) => {
-        resetUsdPrices(debouncedQueue)
-
-        return Promise.reject(error)
```

Comment:
who is setting loading to false now?

Reply:
okay, you just assume it will always succeed cause you do a catch now in processQueue